docker/lib/dependabot/docker/file_fetcher.rb

```diff
@@ -22,27 +22,18 @@ def self.required_files_message
 
       private
 
-      def kubernetes_enabled?
-        Experiments.enabled?(:kubernetes_updates)
-      end
-
       def fetch_files
         fetched_files = []
         fetched_files += correctly_encoded_dockerfiles
-        fetched_files += correctly_encoded_yamlfiles if kubernetes_enabled?
+        fetched_files += correctly_encoded_yamlfiles
```

Comment:
Is there some ruby idiom for why it's instantiated as an empty array rather than `fetched_files = correctly_encoded_dockerfiles + correctly_encoded_yamlfiles`? Is it just trying to guarantee `fetched_files` will never be `nil`?

Reply:
I think this is harmless. I would guess it was stylistic—it accommodates adding in another array on one line without having to reformat.